<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>625</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.483, y1:0, x2:0.472637, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(0, 40, 62, 255));
border-radius:25px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 25pt &quot;Arial&quot;;
border:None</string>
       </property>
       <property name="text">
        <string>Predicciones</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="nueva_prediccion">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>510</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}
QPushButton:hover{
	background-color: rgb(255, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}</string>
       </property>
       <property name="text">
        <string>Nueva Predicci√≥n</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nueva_prediccion_2">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>510</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0,0%);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	border:None;
}
QPushButton:hover{
	background-color: rgb(255, 255, 0);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tabla_predicciones">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>81</y>
         <width>491</width>
         <height>411</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="mostrar_imagen">
       <property name="geometry">
        <rect>
         <x>524</x>
         <y>85</y>
         <width>461</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 75 25pt &quot;Arial&quot;;
border:None</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nueva_prediccion_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
